step/src/main/java/com/google/sps/Message.java

```diff
@@ -25,12 +25,14 @@
   private final String recipientID; 
   private final String text; 
   private final long timestamp;
+  private final String currTime;
 
-  public Message(String senderID, String recipientID, String text, long timestamp) {
+  public Message(String senderID, String recipientID, String text, long timestamp, String currTime) {
       this.senderID = senderID;
       this.recipientID = recipientID;
       this.text = text;
       this.timestamp = timestamp;
+      this.currTime = currTime;
```

Comment:
I don't think it's a good idea to call this variable currTime. It's confusing in the context of something like Message.getCurrTime(). Calling it like timeStamp or something that doesn't imply present-tense seems better to me.